scss/_settings_placeholders.scss

```diff
@@ -2,10 +2,20 @@
 @import 'settings_colors';
 
 // Global placeholder settings
-$input-border-thickness: 1.5px;
```

Comment:
Thanks for going extra mile and implementing it as a CSS variable.

But for simplicity and consistency with color variables, I think we can keep the `$input-border-thickness` SCSS variable, just make it use the value of CSS variable.

So in places where we use it it won't need to change.

To do that I think you just need to define its value as CSS variable:


```
$input-border-thickness: var(--input-border-thickness);
```

And this should be it. All places that currently use `$input-border-thickness` variable will use it as CSS var automatically without need to change their code to explicitly do that.

Reply:
That makes sense! I'll look into changing that this week. :)

Reply:
Code looks good, but we need to move it elsewhere.

The `settings` part should only define SCSS variables, not produce any CSS. By adding CSS variables here we break this convention.

In `settings_colors.scss` we worked around that by defining mixins with CSS variables, but we use them ("render" to CSS) in `base_themes.scss`. I guess we need some more generic "base variables" file for things like that I guess?… 

I'll try to come up with a proposal, if I'm able I'll push into this branch.